### PR TITLE
Added dancer standard finish buff/aura

### PR DESCRIPTION
**Pull request by tsposato**
first time using git etc. so probably needs to be reviewed...

### CODE REVIEW

File: resources/lang/de.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ class CactbotLanguageDe extends CactbotLanguage {
       FoeRequiem: 'Requiem Der Feinde', // up 0x8b, down 0x8c
       LeadenFist: 'Verbesserter Schlag auf Schlag',
       TechnicalFinish: 'Komplexes Finale',
+      StandardFinish: 'FIXME',
```
Comment (quisquous) [inline resources/lang/de.js]: Can you add these translations? https://xivapi.com/Action/16003?pretty=true&columns=Name,Name_de,Name_en,Name_fr,Name_ja

**Comment by quisquous** [comment]
Oh, also if you're up for adding devilment (probably needs to be pretty high on the buff list, since it's 20% crit), feel free to add that too.

**Comment by tsposato** [comment]
> Oh, also if you're up for adding devilment (probably needs to be pretty high on the buff list, since it's 20% crit), feel free to add that too.

Ok done, how is that?

**Comment by quisquous** [comment]
Looks great, thanks a bunch!!

**Comment by haasn** [comment]
Should this perhaps be hidden by default for non-dancers? Not much use to track a buff that has 100% uptime.

**Comment by quisquous** [comment]
It only applies if you get the effect yourself.

**Pull request merged**